security-proxy/README.md

```diff
@@ -3,20 +3,77 @@
 The security-proxy (aka SP) belongs to geOrchestra core, since it is the component which :
  * handles user sessions
  * routes requests to webapps (except CAS)
- 
+
 The behavior is controlled by the files from the `<datadir_root>/security-proxy` folder, which can be found [here](https://github.com/georchestra/datadir/tree/master/security-proxy)
 
+## How-to integrate a new application in geOrchestra ?
+
+The goal here is to take advantage of the [SSO](https://en.wikipedia.org/wiki/Single_sign-on) feature for the new application without having to "CASify" it.
+
+### Proxy configuration
+
+It may sound obvious, but the new application has to be proxified by the security-proxy first !
+This can be configured in the [targets-mapping](https://github.com/georchestra/datadir/blob/master/security-proxy/targets-mapping.properties) file, which can be found in geOrchestra datadir. Remember: changes to this file require the security proxy to be restarted.
+This file maps public URLs to internal (private) URLs.
+For instance, the `header=http://localhost:8280/header/` line means "requests hitting the /header path should be routed to http://localhost:8280/header".
+
+Now, imagine your application has a public frontend (`/newapp/frontend`) and a private backend (`/newapp/backend`).
+You probably would like to restrict backend access to administrators, or people having a specific role.
+
+This can be done very easily with the [security-mappings](https://github.com/georchestra/datadir/blob/master/security-proxy/security-mappings.xml) file, also from the geOrchestra datadir.
+In this file, the last line starting with `<intercept-url pattern=".*" access="IS_AUTHENTICATED_ANONYMOUSLY,ROLE_USER...` means "by default, grant access to every path for anonymous and authenticated users". This means the frontend app is already available to every user, once proxified.
+
+To restrict access to the backend, one just has to insert a new line before the last one.
+Here we choose to restrict backend access to people having the `ADMINISTRATOR` role:
+```xml
+<intercept-url pattern="/newapp/backend" access="ROLE_ADMINISTRATOR" />
+```
+
+It is also possible to create a specific role which grants access to the backend, eg with role `NEWAPP_ADMIN`. The rule becomes:
+```xml
+<intercept-url pattern="/newapp/backend" access="ROLE_NEWAPP_ADMIN" />
+```
+
+### Application configuration
+
+#### Headers
+
+With every request, the proxy adds specific HTTP headers, so the application knows:
```

Comment:
so the application knows: ) tu veux dire que l'application detient alors une information "qu'elle est informée"? si oui : "allowing the application to know : "